Lib/Xamarin.Forms.OpenTok.iOS/Service/PlatformOpenTokService.cs

```diff
@@ -140,15 +162,25 @@ private void OnPropertyChanged(object sender, PropertyChangedEventArgs e)
                         return;
                 }
             }
-            if(SubscriberKit != null)
+            if (Subscribers?.Count > 0)
             {
-                switch(e.PropertyName)
+                switch (e.PropertyName)
                 {
                     case nameof(IsVideoSubscriptionEnabled):
-                        SubscriberKit.SubscribeToVideo = IsVideoSubscriptionEnabled;
+                        //TODO: This is probably wrong.
+                        foreach (OTSubscriber subscriberKit in Subscribers.Select(subscriber => subscriber.Value))
+                        {
+                            subscriberKit.SubscribeToVideo = IsVideoSubscriptionEnabled;
+                        }
+
                         return;
                     case nameof(IsAudioSubscriptionEnabled):
-                        SubscriberKit.SubscribeToAudio = IsAudioSubscriptionEnabled;
+                        //TODO: This is probably wrong.
```

Comment:
Hm, doesn't it work?

Reply:
You should set IsAudioSubscriptionEnabled to false for hitting this breakpoint, I believe